core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -477,7 +477,7 @@ class KafkaApis(val requestChannel: RequestChannel,
           val exceptionsSummary = mergedResponseStatus.map { case (topicPartition, status) =>
             topicPartition -> status.error.exceptionName
           }.mkString(", ")
-          info(
+          error(
```

Comment:
Is this message duplicate to https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/KafkaApis.scala#L613 ? If so, is it better to remove one of them?